packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -74,48 +121,74 @@ async function ensureInitialized() {
   await initializedPromise;
 }
 
+/**
+ * Creates a new {@link EdDSAPCD} by generating a {@link EdDSAPCDProof}
+ * and deriving a {@link EdDSAPCDClaim} from the given {@link EdDSAPCDArgs}.
+ * @param args The set of arguments to make a new {@link EdDSAPCD}.
+ * @returns An instance of the EdDSA PCD.
+ */
 export async function prove(args: EdDSAPCDArgs): Promise<EdDSAPCD> {
   await ensureInitialized();
 
-  if (!args.privateKey.value) {
+  let message;
+
+  if (!args.privateKey.value)
     throw new Error("No private key value provided");
-  }
 
-  if (!args.message.value) {
+  if (!args.message.value)
     throw new Error("No message value provided");
-  }
-
-  let message;
+  
   try {
+    // Converts the list of stringified big integers of the message to a list of big integers.
+    // The reason there is a try/catch around it is to prevent users from passing in
+    // anything other than stringified big integers to sign.
     message = args.message.value.map((fieldStr: string) => BigInt(fieldStr));
   } catch (e) {
     throw new Error("Could not convert message contents to bigint type");
   }
 
+  // Retrieves the id from the arguments or creates a new random id.
   const id = typeof args.id.value === "string" ? args.id.value : uuid();
-
   const prvKey = fromHexString(args.privateKey.value);
 
   const hashedMessage = poseidon(message);
   const publicKey = await getEdDSAPublicKey(prvKey);
+
+  // Make the signature on the message.
   const signature = toHexString(
     eddsa.packSignature(eddsa.signPoseidon(prvKey, hashedMessage))
   );
 
   return new EdDSAPCD(id, { message, publicKey }, { signature });
 }
 
+/**
+ * Verifies an EdDSA PCD by checking that its {@link EdDSAPCDClaim} corresponds to its {@link EdDSAPCDProof}.
+ * @param pcd The {@link EdDSAPCD} to be verified.
+ * @returns true if the {@link EdDSAPCDClaim} corresponds to the {@link EdDSAPCDProof}, otherwise false.
+ */
```

Comment:
these too, with the caveat that you could explain what `true` and `false` represent in line 166